src/Utilities.hs

```diff
@@ -252,3 +254,6 @@ putInfo t = putProgressInfo =<< renderAction
     digest [] = "none"
     digest [x] = x
     digest (x:xs) = x ++ " (and " ++ show (length xs) ++ " more)"
+
+contextDependencies' :: Context -> Action [Context]
```

Comment:
I don't think you need this function. What you need to do is to modify `stage0Packages` in `Settings/Default.hs` removing `terminfo` in case of cross compilation. Then it will automatically be removed from `contextDependencies`.

Reply:
Oh, I was trying to find something like that. Thank!